SPECS/sympa-6.2.spec.in

```diff
@@ -425,24 +425,6 @@ function create_config {
         ## create site configurations
         cp -p %{_sysconfdir}/sympa/data_structure.current_version \
             %{_sysconfdir}/sympa/data_structure.version
-        ## Hide sample ``intranet'' choices.
-        # create_list.conf
-        if [ '!' -e %{_sysconfdir}/sympa/create_list.conf ]; then
-            cp -p %{_datadir}/sympa/default/create_list.conf \
-                %{_sysconfdir}/sympa/create_list.conf
-            cd %{_datadir}/sympa/default/create_list_templates
-            for template in *intranet*; do
-                ( echo ''; echo "$template hidden" ) \
-                    >> %{_sysconfdir}/sympa/create_list.conf
-            done
-        fi
-        # scenari
-        cd %{_datadir}/sympa/default/scenari
-        for scenario in *intranet*; do
-            if [ '!' -e %{_sysconfdir}/sympa/scenari/$scenario ]; then
-                touch %{_sysconfdir}/sympa/scenari/$scenario:ignore
-            fi
-        done
```

Comment:
I guess the removal should be done upstream, I've filed an issue about this : 
https://github.com/sympa-community/sympa/issues/119
Meanwhile, I'll also remove the intranet related files (default/create_list_templates/intranet_list and default/scenari/*intranet*) in %install, which should have the same effect than the removed code.
(it would be better to remove the files in %prep, but then configure complains, as the Makefile needs to be tweaked too)